Dwolla.Client/HttpServices/DwollaHttpService.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Dwolla.Client.HttpRequestServices;
+using Dwolla.Client.HttpServices.Architecture;
+using Dwolla.Client.Models;
+using Dwolla.Client.Models.Responses;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Dwolla.Client.HttpServices
+{
+	public class DwollaHttpService
+	{
+		private IDwollaClient _client;
+		public IDwollaClient Client
+		{
+			get => _client ?? DwollaConfiguration.Client;
+			set => _client = value;
+		}		
+		
+		private static TokenResponse _cachedToken;
+
+		private Task<string> getAccessTokenAsync;	
```

Comment:
Private vars should still have leading `_` in name even though they're functions